miditools/src/main/java/com/mobileer/miditools/MidiPortSelector.java

```diff
@@ -56,13 +56,13 @@ public MidiPortSelector(MidiManager midiManager, Activity activity,
         mMidiManager = midiManager;
         mActivity = activity;
         mType = type;
-        mAdapter = new ArrayAdapter<MidiPortWrapper>(activity,
+        mSpinner = (Spinner) activity.findViewById(spinnerId);
+        mAdapter = new ArrayAdapter<>(mSpinner.getContext(),
```

Comment:
Yes no problem with that.
It's not required. Not sure if the compiler add it back when generating bytecode. Anyway it's deducted from the mAdapter declaration.